packages/database/test/query.test.ts

```diff
@@ -1667,6 +1891,64 @@ describe('Query Tests', () => {
     );
   });
 
+  it('Ensure startAt / endBefore with priority and name works (2).', () => {
```

Comment:
If you need more things on your plate, can you also use a different suffix for this test name?